### ISSUE TITLE
cartoon way off the model

### ISSUE THREAD

**Issue by fragalysis-reporter** [m2ms/fragalysis-frontend#415]
- Version: 0.8.8
- Name: Anthony Aimon
- Email: anthony.aimon@diamond.ac.uk
- Description: I have not been able to zoom in and use the pen to make it more obvious. x0282 pyrimidine ring looks too close to the ribbon. When selecting the actual model there is a big difference between the position of the ribbon and the model of the protein structure.
It might be diffuclt to have a averaged ribbon that would look ok for all structures - but maybe an option to select a more suitable ribbon? Especially when you only look at one bound ligand. To discuss?
- Reducers: https://github.com/m2ms/fragalysis-assets/blob/master/Reducers-1594634430824638295.json?raw=false

![screenshot](https://github.com/m2ms/fragalysis-assets/blob/master/Screenshot-1594634421608960640.png?raw=true)

**Comment by reskyner** [comment]
Good idea - we should add an option in fragalysis (even a drop-down) to change which ribbon is being used as the reference. 

We should also do something smarter - like select the reference by lowest RMSD on alignment to all other protein backbones or something
